File: Daily Challanges/2026-05-May/2026-05-25.js
function guessNumber(secret, guess) {
  return secret === guess ? "you got it!" : secret > guess ? "higher" : "lower";
}

console.log(guessNumber(50, 30));
console.log(guessNumber(85, 99));
console.log(guessNumber(2026, 2026));
console.log(guessNumber(2026, 2026));

File: Daily Challanges/2026-05-May/2026-05-25.py
def guess_number(secret, guess):
    return "higher" if secret > guess else "lower" if secret < guess else "you got it!"

print(guess_number(50, 30))
print(guess_number(85, 99))
print(guess_number(2026, 2026))
print(guess_number(92904, 11283))